app/src/main/java/org/jellyfin/androidtv/details/DetailsOverviewPresenter.kt

```diff
@@ -119,34 +122,36 @@ class DetailsOverviewPresenter : RowPresenter() {
 			viewHolder.subtitle.visibility = View.GONE
 		}
 
-		// rating
-		if (item is Movie) { //todo move those properties to baseitem or something
+		if (item is Movie) {
 			if (item.productionYear != null) {
 				viewHolder.year.text = item.productionYear.toString()
 				viewHolder.year.visibility = View.VISIBLE
-			} else {
-				viewHolder.year.visibility = View.GONE
 			}
+		}
 
+		if (item is Episode) {
+			if (item.premiereDate != null) {
+				val format = DateFormat.getDateFormat(context)
+				viewHolder.year.text = format.format(item.premiereDate)
```

Comment:
Assuming this will display month & day too I'd suggest renaming "year" to "premiere" or something similar to avoid confusion.